docs/core/base/document.md

```diff
@@ -0,0 +1,112 @@
+# Document
+
+The term "Document" is used in the only in the code. It refers a single piece of content that created by Elementor, such as a post, page, or a saved template.
+
```

Comment:
I think you should firstly explain what a Document is and why it exists, and then say that it's only a code-term